app/(home)/page.tsx

```
@@ -6,17 +7,53 @@ import FAQSection from '@/app/(home)/_section/FAQSection';
-      <Header />
-      <HeroSection />
-      <ServiceFeaturesSection />
-      <ReviewSection />
-      <FAQSection />
-      <CTASection />
-      <FloatingButton />
-    </PageBackground>
+    <>
+      <script
+        type='application/ld+json'
+        dangerouslySetInnerHTML={{ __html: JSON.stringify(structuredData) }}
```

Biome (2.4.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

(lint/security/noDangerouslySetInnerHtml)